js/data.json

```diff
@@ -827,7 +827,7 @@ osts = [
 	},
 
 	{
-		"link" : "http://66.90.93.122/ost/naruto-shippuden-original-soundtrack-3/ygmuwqzvwj/08.-ashura-indra.mp3",
+		"link" : "https://www.youtubeinmp3.com/download/get/?i=9kEnefIYeIqtlqU4BlSYsPcD3lWoh3SZ&e=92&v=8-tyjXQn-Cw&progressType=button&color=008000",
```

Comment:
This link never returns a mp3 file, __For links of YoutubeinMP3 to return mp3 files, change your download to fetch__: 
For eg. here change your link to 
`https://www.youtubeinmp3.com/fetch/?video=https://www.youtube.com/watch?v=8-tyjXQn-Cw`
and also see that your current link never return a valid mp3, so you also shouldn't tick : 
- [ ] I have checked my addition locally and it is working and won't break existing entries.
###### PS - There may be some intentions of mine for writing large comments.